apps/imaginary.sh

```diff
@@ -116,16 +123,28 @@ nextcloud_occ config:system:set preview_concurrency_new --value="4"
 
 # Set providers (https://github.com/nextcloud/server/blob/master/lib/private/Preview/Imaginary.php#L60)
 # https://github.com/nextcloud/vm/pull/2464#discussion_r1155074227
-# This is handled by Imagniary itself
-nextcloud_occ config:system:set enabledPreviewProviders 0 --value="OC\\Preview\\Imaginary"
+nextcloud_occ config:system:set enabledPreviewProviders 1 --value="OC\\Preview\\BMP"
+nextcloud_occ config:system:set enabledPreviewProviders 2 --value="OC\\Preview\\x-bitmap"
+nextcloud_occ config:system:set enabledPreviewProviders 3 --value="OC\\Preview\\PNG"
+nextcloud_occ config:system:set enabledPreviewProviders 4 --value="OC\\Preview\\JPEG"
+nextcloud_occ config:system:set enabledPreviewProviders 5 --value="OC\\Preview\\GIF"
+nextcloud_occ config:system:set enabledPreviewProviders 6 --value="OC\\Preview\\HEIC"
+nextcloud_occ config:system:set enabledPreviewProviders 7 --value="OC\\Preview\\HEIF"
+nextcloud_occ config:system:set enabledPreviewProviders 8 --value="OC\\Preview\\SVG"
+nextcloud_occ config:system:set enabledPreviewProviders 9 --value="OC\\Preview\\TIFF"
+nextcloud_occ config:system:set enabledPreviewProviders 10 --value="OC\\Preview\\WEBP"
+nextcloud_occ config:system:set enabledPreviewProviders 11 --value="OC\\Preview\\XML"
+nextcloud_occ config:system:set enabledPreviewProviders 12 --value="OC\\Preview\\PDF"
+nextcloud_occ config:system:set enabledPreviewProviders 13 --value="OC\\Preview\\Ilustrator"
+nextcloud_occ config:system:set enabledPreviewProviders 14 --value="OC\\Preview\\Movie"
+nextcloud_occ config:system:set enabledPreviewProviders 15 --value="OC\\Preview\\Imaginary"
```

Comment:
As I said only the ones that are not covered by imaginary need to get enabled:

```suggestion
nextcloud_occ config:system:set enabledPreviewProviders 0 --value="OC\\Preview\\Imaginary"
nextcloud_occ config:system:set enabledPreviewProviders 1 --value="OC\\Preview\\Image"
nextcloud_occ config:system:set enabledPreviewProviders 2 --value="OC\\Preview\\MarkDown"
nextcloud_occ config:system:set enabledPreviewProviders 3 --value="OC\\Preview\\MP3"
nextcloud_occ config:system:set enabledPreviewProviders 4 --value="OC\\Preview\\TXT"
nextcloud_occ config:system:set enabledPreviewProviders 5 --value="OC\\Preview\\OpenDocument"
nextcloud_occ config:system:set enabledPreviewProviders 6 --value="OC\\Preview\\Movie"
```

Reply:
But there are more than you added...?

return '/(image\/(bmp|x-bitmap|png|jpeg|gif|heic|heif|svg\+xml|tiff|webp)|application\/(pdf|illustrator))/'; Shouldn't those be added as well?